implementations/python/provekit-lift-python-source/src/provekit_lift_python_source/verify_rpc.py

```
@@ -0,0 +1,267 @@
+    return "bare"
+
+
+def _iter_py_files(root: Path):
+    for dirpath, dirnames, filenames in os.walk(root):
+        dirnames[:] = [d for d in dirnames if d not in _IGNORED_DIRS]
+        for filename in sorted(filenames):
+            if filename.endswith(".py"):
+                yield Path(dirpath) / filename
+
+
+def lift_workspace(root: str, mode: str) -> tuple[list[Json], list[Json]]:
```

Ruff (0.15.13)

[error] Undefined name `Json`

(F821)